decisions/mnm_society_decisions.txt

```diff
@@ -2717,10 +2709,13 @@ targeted_decisions = {
 			ai = yes
 			NOT = { character = FROM }
 			is_patrician = no
+			
+			# Warcraft
 			OR = {
 				lower_tier_than = DUKE
-				is_landed = no
+				is_ruler = no
```

Comment:
Don't forget to put in change log that you fixed vanilla bug, since your character could abduct landless character above duke tier before.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75931989-5c476480-5e87-11ea-8846-0b0cbd458fac.png)

</details>

Reply:
Mentioned in the scripting changelog.